prow/config.yaml

```diff
@@ -1302,6 +1302,62 @@ presubmits:
           hostPath:
             path: /mnt/disks/ssd0
 
+  - name: pull-kubernetes-e2e-gce-dind-ipv6
+      agent: kubernetes
+      context: pull-kubernetes-e2e-gce-dind-ipv6
+      max_concurrency: 8
+      skip_report: true
+      run_if_changed: '^(cmd/kubeadm|build/debs).*$'
+      rerun_command: "/test pull-kubernetes-e2e-gce-dind-ipv6"
+      trigger: "(?m)^/test pull-kubernetes-e2e-gce-dind-ipv6,?(\\s+|$)"
+      skip_branches:
+      - release-1.6 # doesn't support IPv6
+      - release-1.7 # doesn't support IPv6
+      - release-1.8 # doesn't support IPv6
+      spec:
+        containers:
+        - image: gcr.io/k8s-testimages/e2e-gce-dind:v20180322-269013b-dirty
+          args:
+          - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+          - "--upload=gs://kubernetes-jenkins/pr-logs"
+          - "--git-cache=/root/.cache/git"
+          - "--timeout=75"
+          - "--clean"
+          env:
+          - name: USER
+            value: prow
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/service-account/service-account.json
+          - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+            value: /etc/ssh-key-secret/ssh-private
+          - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+            value: /etc/ssh-key-secret/ssh-public
+          - name: SKIP_RELEASE_VALIDATION # See: https://github.com/kubernetes/kubernetes/pull/50391
+            value: true
+          volumeMounts:
+          - name: service
+            mountPath: /etc/service-account
+            readOnly: true
+          - name: ssh
+            mountPath: /etc/ssh-key-secret
+            readOnly: true
+          - name: cache-ssd
+            mountPath: /root/.cache
```

Comment:
please remove this mount and the containerPort, I've just finished eliminating this.

Reply:
Will remove these, thanks.